src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml

```diff
@@ -10,4 +10,7 @@
   <include file="remove_items_holdingsrecord_instance_title_searchable.xml" relativeToChangelogFile="true"/>
   <include file="update_item_holdingsrecord_entity_type_definition.xml" relativeToChangelogFile="true"/>
 
+  <include file="fix_user_details_preferred_contact_type.xml" relativeToChangelogFile="true"/>
```

Comment:
The changes required to User preferred contact type consistent are achieved by including below two files which indeed needs to be added in the changeling